# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/mtools/devfs.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./floppyd.c.orig	Tue Oct 26 14:51:02 1999
+++ ./floppyd.c	Tue Oct 26 14:51:48 1999
@@ -894,7 +894,7 @@
 	}
 	
 	if(!device_name) {
-		device_name = "/dev/fd0";
+		device_name = "/dev/floppy/0";
 	}
 
 	if (!device_name || !*device_name) {
--- ./devices.c.orig	Thu May 11 13:58:15 2000
+++ ./devices.c	Thu May 11 14:02:17 2000
@@ -709,12 +709,8 @@
 
 #define predefined_devices
 struct device devices[] = {
-	{"/dev/fd0", 'A', 0, O_EXCL, 80,2, 18,0, MDEF_ARG},
-	{"/dev/fd1", 'B', 0, O_EXCL, 0,0, 0,0, FDEF_ARG},
-	/* we assume that the Zip or Jaz drive is the second on the SCSI bus */
-	{"/dev/sdb4",'J', GENHD },
-	{"/dev/sdb4",'Z', GENHD },
-	/*	{"/dev/sda4",'D', GENHD },*/
+	{"/dev/floppy/0", 'A', 0, O_EXCL, 80,2, 18,0, MDEF_ARG},
+	{"/dev/floppy/1", 'B', 0, O_EXCL, 0,0, 0,0, FDEF_ARG},
 	REMOTE
 };
 
